### PR TITLE
Add eject quotes

### PR DESCRIPTION
**Pull request by JMM889901**
Lets modders add extra eject messages using AddCommonEjectMessage and AddRareEjectMessage
Yep

### CODE REVIEW

**Comment by GeckoEidechse** [comment]
We really need a way to separate Respawn squirrel vs contributor modified stuff ^^

**Comment by uniboi** [comment]
Would be easiest if or authors would just commit the vanilla script before doing their changes

**Comment by JMM889901** [comment]
> Would be easiest if or authors would just commit the vanilla script before doing their changes

I did, the first commit is upload from scripts, everything past that is modified

**Comment by GeckoEidechse** [comment]
Aight, that's good then. I guess the best way to do it though, as we do commit squashing, is once the PR is accepted, instead of merging it directly, have someone merge the vanilla Squirrel stuff first with `Respawn` as author and then merge this PR.

For anyone reviewing this PR, use the following link to see the actual changes made:

https://github.com/R2Northstar/NorthstarMods/pull/527/files/3af658dab2d795d23e72ad7feaec7a5a60e9e0ca..23a38820b42821da0788187637720128e398ae6f
(Replace `23a38820b42821da0788187637720128e398ae6f` with the hash of the newest commit in the PR in case of additional changes from the time of writing)

**Review by uniboi — APPROVED** [review]
Code looks like it works

File: Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut
Comment (ASpoonPlaysGames) [inline Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut:1035]:
```suggestion
		if ( index <= COCKPIT_EJECT_RARE_COUNT )
```

**Comment by JMM889901** [comment]
Had to do some of the suggestions manually because github was being difficult

**Comment by ASpoonPlaysGames** [comment]
Gonna re-review this tomorrow when I get home but github doesnt want me to be able to request a review from myself so here we are

**Comment by JMM889901** [comment]
Github moment

File: Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut
Comment (ASpoonPlaysGames) [inline Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut:1045]: In the event that `index == COCKPIT_EJECT_RARE_COUNT` this code will try to return an ejection message that doesn't exist (`"#COCKPIT_EJECT_RARE_36"`) instead of returning `file.moddedRareEjectMessages[0]`
This also means that `file.moddedRareEjectMessages[0]` will never be used.

image


```suggestion
		if ( index < COCKPIT_EJECT_RARE_COUNT )
			return "#COCKPIT_EJECT_RARE_" + index
		else
			return file.moddedRareEjectMessages[index - COCKPIT_EJECT_RARE_COUNT]
```
Also {} isnt needed for single line things, neither is the else technically but personally I prefer to keep it for better readability

File: Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut
Comment (ASpoonPlaysGames) [inline Northstar.Client/mod/scripts/vscripts/client/cl_titan_cockpit.nut:1053]:
```suggestion
	if ( index < COCKPIT_EJECT_COMMON_COUNT )
		return "#COCKPIT_EJECT_COMMON_" + index
	else
		return file.moddedCommonEjectMessages[index - COCKPIT_EJECT_COMMON_COUNT]
```
Same as above

**Review by ASpoonPlaysGames — CHANGES_REQUESTED** [review]

**Pull request merged**
